content/en/logs/log_collection/javascript.md

```diff
@@ -1261,3 +1262,4 @@ window.DD_LOGS && window.DD_LOGS.getInternalContext() // { session_id: "xxxx-xxx
 [13]: https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted
 [14]: /integrations/content_security_policy_logs/#use-csp-with-real-user-monitoring-and-session-replay
 [15]: #user-tracking-consent
+[16]: https://docs.datadoghq.com/data_security/logs/#pci-dss-compliance-for-log-management
```

Comment:
**🥜 nitpick:** ‏I think we might want to use a relative link here? (Not sure if it matters)

```suggestion
[16]: /data_security/logs/#pci-dss-compliance-for-log-management
```